vllm/v1/core/scheduler.py

```diff
@@ -607,8 +624,10 @@ def _check_stop(self, request: Request) -> bool:
         return False
 
     def add_request(self, request: Request) -> None:
+        tokens = self._get_request_total_tokens(request)
         self.waiting.append(request)
         self.requests[request.request_id] = request
+        self._waiting_tokens += tokens
```

Comment:
Minor: `self._waiting_tokens += self._get_request_total_tokens(request)` like the line below?